webapp/publisher/snaps/logic.py

```diff
@@ -310,3 +328,27 @@ def convert_date(date_to_convert):
     """
     date_parsed = parser.parse(date_to_convert).replace(tzinfo=None)
     return date_parsed.strftime("%B %Y")
+
+
+def categorise_media(media):
```

Comment:
Do we have a test for it?

Reply:
I cover it in the listing page post and get endpoint testing

Reply:
https://codecov.io/gh/canonical-websites/snapcraft.io/pull/1696/diff?src=pr&el=tree#D2-329